src/Connection/ConnectionInterface.php

```diff
@@ -228,4 +229,16 @@ public function setTablePrefix(string $value): void;
      * @psalm-param Closure(ConnectionInterface):mixed|Closure(ConnectionInterface):void $closure
      */
     public function transaction(Closure $closure, ?string $isolationLevel = null): mixed;
+
+    /**
+     * Returns maximum number of bound params for a DBMS. Default 0 - means unlimited.
```

Comment:
```suggestion
     * Returns maximum number of bound params for a DBMS. 0 means unlimited.
```

Interface don't define default value.